tests/test_read_database.py

```diff
@@ -0,0 +1,82 @@
+from architxt.database import read_table, read_unreferenced_table
+from architxt.tree import Tree
+from sqlalchemy import create_engine, MetaData, Table, Column, Integer, String, ForeignKey
+from sqlalchemy.orm import sessionmaker
+
+
+def create_test_database(engine: create_engine) -> None:
```

Comment:
Use PyTest fixture for test setup : https://docs.pytest.org/en/6.2.x/fixture.html

The session should also be provided by another fixture